Answers.txt

```diff
@@ -1,7 +1,29 @@
 • What is a gitignore file and what should it include for a Java project? 
 
-• What does fast-forward in git mean? 
+Gitignore file is used for ignoring certain types of files during commit. 
```

Comment:
Do the gitignore files get pushed when we do a git push?

Reply:
No, the main reason for using gitignore file is to ignore these files during commit and push

Reply:
okay